util/argo/argo.go

```diff
@@ -422,8 +422,8 @@ func GetAppProject(app *argoappv1.Application, projLister applicationsv1.AppProj
 		return nil, err
 	}
 	if !proj.IsAppNamespacePermitted(app, ns) {
-		return nil, fmt.Errorf("application '%s' in namespace '%s' is not allowed to use project '%s'",
-			app.Name, app.Namespace, proj.Name)
+		return nil, fmt.Errorf("application '%s' is not allowed to use project '%s'",
+			app.Name, proj.Name)
```

Comment:
```suggestion
		return nil, fmt.Errorf("application '%s' is not allowed to use project '%s'",
			app.QualifiedName(), proj.Name)
```

Reply:
IMHO, the namespace is integral part of the error message.

Using `app.QualifiedName()` will result in a `<namespace>/<name>` string, so the error message would be:

`"application 'foo/bar' is not allowed to use project 'baz'"`

Reply:
make sense.

Reply:
Hm, seems you haven't pulled that change in.